setup.cfg

```diff
@@ -0,0 +1,31 @@
+[metadata]
+name = tool
+version = attr: tool.version.__version__
+url = https://github.com/user-cont/tool
+description = Easy way to git.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Red Hat
+author_email = user-cont-team@redhat.com
+license = GPLv3+
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Topic :: Software Development
+keywords = git, github
+
+[options]
+packages = find:
```

Comment:
I do not know this!

Is it possible to also exclude packages (e.g. tests)? (It is only a matter of time to add them.)

Reply:
added commented out section for excluding `tests/`